jobs/sweep_schedulers/README.md

```diff
@@ -0,0 +1,94 @@
+## Sweeps on launch -- custom scheduler jobs
+
+This folder contains jobs that can be used as schedulers for launch sweeps. For examples using jobs that have already been created, head over to the `wandb/examples` repo [here](https://github.com/wandb/examples/tree/master/examples/launch/launch-sweeps).
+
+### Quickstart using premade jobs
+
+To run an example using the Wandb sweep logic, **ensure the target launch queue is active** (more info [here](https://docs.wandb.ai/guides/launch/quickstart)), and run:
+
+```bash
+wandb launch-sweep wandb_scheduler/example_configs/basic.yaml --queue <queue> --project <project> --entity <entity>
+```
+
+This basic config contains a few example parameters to get a scheduler running, shown below: 
+
+```yaml
+# basic.yaml
+scheduler:
+   job: 'wandb/jobs/Wandb Scheduler Image Job:latest'
+   resource: local-container  # required for image jobs
+
+   settings:
+      method: bayes  # required to specify method here for Wandb scheduler 
+
+job: 'wandb/jobs/Example Train Job:latest'
+run_cap: 5
+metric:
+  goal: maximize
+  name: val_acc
+
+parameters:
+   param1:
+      min: 0
+      max: 10
+```
+
+Or, to run the pre-made Optuna Scheduler job, run: 
+
+```bash
+wandb launch-sweep optuna_scheduler/example_configs/basic.yaml --queue <queue> --project <project> --entity <entity>
+```
+
+More information specific to the Optuna sweep scheduler can be found in the `wandb/examples` repo [here](https://github.com/wandb/examples/tree/master/examples/launch/launch-sweeps/optuna-scheduler)
+
+### Custom scheduler jobs
+
+While the above example uses a pre-made scheduler job (`'wandb/jobs/Wandb Scheduler Image Job:latest'`), they can also be created using this repo. Clone the repo, modifying the `*_scheduler` scripts with impunity, and then create jobs from them, with:
+
+`python wandb_scheduler/wandb_scheduler.py` or `python optuna_scheduler/optuna_scheduler.py`
+
+For a specific example of a simple modification to the `wandb_scheduler.py` file, check out the example in `wandb/examples` [here](https://github.com/wandb/examples/tree/master/examples/launch/launch-sweeps/custom-scheduler).
+
+Note: There are three possible job-types that can be created from this script, in the following ways:
```

Comment:
Wording is weird here, maybe:
Note: there are 3 possible job source types that one can use to create a job: